### PR TITLE
Skip Video demo in integration tests of flutter_gallery

### PR DESCRIPTION
**Pull request by zanderso**
Context in https://github.com/flutter/flutter/issues/100497

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Why skip all of the video demo rather than just removing the network-based demo? There's still the asset-based video.

**Comment by zanderso** [comment]
> Why skip all of the video demo rather than just removing the network-based demo? There's still the asset-based video.

I'm afraid I don't have the cycles to whip out the scalpel and make a finer-grained change. I'm fine with closing this PR if retaining all of the other demos in the benchmark is important.

**Review by stuartmorgan-g — APPROVED** [review]
Makes sense; LGTM as a short-term fix

File: dev/integration_tests/flutter_gallery/test_driver/run_demos.dart [hunk 1]
```diff
@@ -12,9 +11,9 @@ import 'package:flutter_test/flutter_test.dart';
 /// Demo names are formatted as 'DEMO_NAME@DEMO_CATEGORY' (see
 /// `demo_lists.dart` for more examples).
 final List<String> kSkippedDemos = <String>[
-  // The CI uses Chromium, which lacks the video codecs to run this demo.
```
[the remark this answers is not in the document]
Comment (ditman) [reply dev/integration_tests/flutter_gallery/test_driver/run_demos.dart:15]: Yes, this helped a lot with web + chromium (96): https://github.com/flutter/plugins/pull/5068/commits/f3b5740f867d12a87458fb6227d867f8ab6bc60c

**Pull request merged**